package.json

```diff
@@ -34,7 +34,8 @@
     "test:unit": "cross-env NODE_NO_WARNINGS=1 jest",
     "test:unit:watch": "npm run test:unit -- --watch",
     "test": "turbo test:unit",
-    "prepare": "husky"
+    "prepare": "husky",
+    "cm": "cz"
```

Comment:
is this just for convenience if someone wants to call it directly? it seems like the primary use case is via husky git hook

Reply:
You're right, the primary use case is with the husky git hook. 

If you want I can remove the script?

Reply:
Go ahead :)

Reply:
Okay! All done.